exporter/zipkin/example_test.go

```diff
@@ -17,24 +17,17 @@ package zipkin_test
 import (
 	"log"
 
-	openzipkin "github.com/openzipkin/zipkin-go"
-	"github.com/openzipkin/zipkin-go/reporter/http"
 	"go.opencensus.io/exporter/zipkin"
 	"go.opencensus.io/trace"
 )
 
 func Example() {
-	// import (
-	//     openzipkin "github.com/openzipkin/zipkin-go"
-	//     "github.com/openzipkin/zipkin-go/reporter/http"
-	//     "go.opencensus.io/exporter/trace/zipkin"
-	// )
-
-	localEndpoint, err := openzipkin.NewEndpoint("server", "192.168.1.5:5454")
+	exporter, err := zipkin.NewExporterWithOptions(&zipkin.Options{
+		EndpointHostPort: "192.168.1.5:5454",
```

Comment:
Endpoint service name is quite significant. Please also add it.

Reply:
Roger that, the current exporter always uses "server" though.

Reply:
`os.Args[0]` is probably a better default

Reply:
The current exporter is not using the "server". We allow users to configure it.